binding/web/core/README.md

```diff
@@ -0,0 +1,16 @@
+# porcupine-web-core
+
+Package containing types for Porcupine web bindings. 
+
+For more details, check [binding/web](binding/web).
```

Comment:
I think the link won't work. can you give it a try?

Reply:
I removed to link since it was in a different directory at start. Now that is is a subdir from the web binding I don't think we need it.